src/ApiHandler.ts

```diff
@@ -400,7 +400,13 @@ export default class ApiHandler {
 
 		return {
 			at,
-			staking,
+			staking: {
```

Comment:
fyi, `Codec`s also have `.toJSON()` and `.toHuman()`, which might suit better than .toString (e.g. for fields that are numbers)

Reply:
+1 for using something on `staking` itself rather than its fields.

Reply:
Does this change the "shape" of the HTTP response?